test/test_pysodium.py

```diff
@@ -61,10 +61,31 @@ def test_crypto_box_seal(self):
         self.assertEqual(pysodium.crypto_box_seal_open(c, pk, sk), b'howdy')
 
     def test_crypto_box_open(self):
+        m = b"howdy"
         pk, sk = pysodium.crypto_box_keypair()
         n = pysodium.randombytes(pysodium.crypto_box_NONCEBYTES)
-        c = pysodium.crypto_box(b"howdy", n, pk, sk)
-        pysodium.crypto_box_open(c, n, pk, sk)
+        c = pysodium.crypto_box(m, n, pk, sk)
+        plaintext = pysodium.crypto_box_open(c, n, pk, sk)
+        self.assertEqual(m, plaintext)
+
+        # Test the easy variants on the same message.
+        c_easy = pysodium.crypto_box_easy(m.decode(), n, pk, sk)
```

Comment:
IMO: it would be better to split this test.
test_crypto_box_open_easy
test_crypto_box_open_afternm
test_crypto_box_open_easy_afternm